Cargo.toml

```diff
@@ -10,3 +10,9 @@ members = [
   "neqo-interop",
   "test-fixture",
 ]
+
+[patch.crates-io]
+#nix = { path = "./../nix" }
+#libc = { path = "./../libc" }
+libc = { git = "https://github.com/larseggert/libc.git", branch = "feat-ttl" }
+nix = { git = "https://github.com/larseggert/nix.git", branch = "feat-tos" }
```

Comment:
We won't be able to land a change that pulls from a private branch, at least not without an extraordinary approval process.  I assume that you are in the process of landing changes upstream.

Reply:
Yep. This is just temporary so the CI runs. I will remove this once upstreams are updated.